#!/usr/bin/env python
PACKAGE = "plane_segment"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

##
gen.add("display_input_cloud",  bool_t, 0,  "", True)
gen.add("display_scanline_cloud",   bool_t, 0, "", False)
gen.add("display_line_cloud",  bool_t, 0,  "", False)
gen.add("display_normal",   bool_t, 0,  "", True)
gen.add("display_normal_arrow", bool_t, 0, "", True)
gen.add("display_plane",    bool_t, 0,  "", True)
gen.add("display_plane_arrow",    bool_t, 0,  "", False)
gen.add("display_plane_inlier", bool_t, 0, "", True)
gen.add("display_plane_projected_inlier",    bool_t, 0,  "", False)
gen.add("display_plane_boundary", bool_t, 0, "", True)
gen.add("display_plane_hull",    bool_t, 0,  "", False)

exit(gen.generate(PACKAGE, "plane_segment", "Viewer"))
